appium/webdriver/extensions/location.py

```diff
@@ -38,14 +38,16 @@ def toggle_location_services(self: T) -> T:
         return self
 
     def set_location(
-        self: T, latitude: Union[float, str], longitude: Union[float, str], altitude: Union[float, str] = None
+        self: T, latitude: Union[float, str], longitude: Union[float, str],
+            altitude: Union[float, str] = None, speed: Union[float, str] = None
     ) -> T:
         """Set the location of the device
 
         Args:
             latitude: String or numeric value between -90.0 and 90.00
             longitude: String or numeric value between -180.0 and 180.0
             altitude: String or numeric value (Android real device only)
+            speed: String or numeric value (Android real devices only)
```

Comment:
could you please add more detailed description of this arg?

Reply:
it also makes sense to mention what is the minimum Appium version for it to be available

Reply:
honestly i don't know what is the minimum Appium version. how do i find that out?

Reply:
this will be the next published one: 1.21

Reply:
i added a bit more details to description but waiting to find out about how to determine the minimum Appium version.